### DISCUSSION TITLE
Use partials globally without relative paths

### DISCUSSION THREAD

**Discussion by codecansdev** [11ty/eleventy#2600] [Q&A]
Hi,

 I am starting to work with eleventy and handlebars migrating a website which used handlebars-loader for webpack [https://github.com/pcardune/handlebars-loader](https://github.com/pcardune/handlebars-loader) and I was wondering if there is a way to allow  partials to be defined in a directory and used globally without relative paths in eleventyjs, like the partialDirs option in **handlebars-loader** allows us to do?

Thank you in advance for your reply,

Paulo

**Comment by pdehaan** [comment]
I don't have much experience w/ Handlebars (generally preferring LiquidJS or Nunjucks), but you might be able to use the https://www.11ty.dev/docs/config/#directory-for-includes directory.

**Reply by codecansdev** [reply]
thank you @pdehaan :) that's actually what I am currently using.. 

I was wondering if there is something that would allow me to organize my partials within folders (i.e. _includes/modules/globals/footer.hbs, _includes/modules/cards/carrousel_card.hbs, etc) and still refer to them globally {{> footer}} {{> carrousel_card}} independent of the directory they are in... that's what [handlebars-loader](https://github.com/pcardune/handlebars-loader) helps me with in my other project using the partialDirs. You can take a look at this example as reference [https://github.com/pcardune/handlebars-loader/tree/main/examples/partialDirs](https://github.com/pcardune/handlebars-loader/tree/main/examples/partialDirs)
